hedera-node/hedera-app/src/main/java/com/hedera/node/app/HederaApp.java

```diff
@@ -119,6 +126,9 @@ interface Builder {
         @BindsInstance
         Builder maxSignedTxnSize(@MaxSignedTxnSize final int maxSignedTxnSize);
 
+        @BindsInstance
+        Builder workingState(NonAtomicReference<HederaState> workingStateRef);
```

Comment:
This can be deleted since you've introduced `WorkingStateAccessor`.

Reply:
Makes sense. Done